File: src/main/java/org/jukeboxmc/potion/StrengthEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class StrengthEffect extends Effect {

    @Override
    public int getId() {
        return 5;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.STRENGTH;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 147, 36, 35 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/StrengthEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class StrengthEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 5
    }

    override fun getEffectType(): EffectType {
        return EffectType.STRENGTH
    }
    
    override fun getEffectColor(): Color {
        return Color(147, 36, 35)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}